### PR TITLE
Add test cases

### PR DESCRIPTION
**Pull request by RunjiaChen**

### CODE REVIEW

File: src/test/java/seedu/address/AddCommandHelperTest.java [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+            a.getResponse("Bishan St 39");
+            a.getResponse(" ");
+            a.getResponse("input here does not  matter");
+        } catch (ParseException e) {
+            assert false : "No exception should be thrown for valid name";
+        }
+        System.out.println(a.getStatus());;
+        assertEquals(a.getStatus(), Status.COMPLETE);
+
+    }
```
Comment (smolegz) [inline src/test/java/seedu/address/AddCommandHelperTest.java:113]: Good job testing edge cases!

**Review by smolegz — APPROVED** [review]
LGTM
